### PR TITLE
Veoss 918

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

File: app/models/SessionData.scala [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+object SessionData {
+
+  val reads: Reads[SessionData] = {
+
+    import play.api.libs.functional.syntax._
+
+    (
+      (__ \ "userId").read[String] and
+        (__ \ "data").read[JsObject] and
```
Comment (owen-opencast) [inline app/models/SessionData.scala:72]: Align indentation

File: app/repositories/SessionRepository.scala [hunk 2]
```diff
@@ -17,56 +17,43 @@
...
-          .name("lastUpdatedIdx")
-          .expireAfter(appConfig.cacheTtl, TimeUnit.SECONDS)
-      ),
-      IndexModel(
-        Indexes.ascending("userId", "period"),
-        IndexOptions()
-          .name("userIdAndPeriodIdx")
-          .unique(true)
-      )
+class SessionRepository@Inject()(
```
Comment (owen-opencast) [inline app/repositories/SessionRepository.scala:32]: Space before "@inject"

**Review by owen-opencast — APPROVED** [review]

File: app/repositories/SessionRepository.scala (hunk 2 above)
Comment (oliwiaKot) [reply app/repositories/SessionRepository.scala]: done

File: app/models/SessionData.scala (hunk 1 above)
Comment (oliwiaKot) [reply app/models/SessionData.scala:72]: done

File: app/controllers/PreviousReturnController.scala [hunk 3]
```diff
@@ -93,12 +97,14 @@ class PreviousReturnController @Inject()(
           totalVatList = Some(totalVatList),
           displayPayNow = displayPayNow,
           vatOwedInPence = vatOwedInPence,
-          displayBanner = displayBanner
+          displayBanner = displayBanner,
+          externalUrl
```
[the remark this answers is not in the document]
Comment (oliwiaKot) [reply app/controllers/PreviousReturnController.scala]: done

File: app/controllers/ContinueReturnController.scala [hunk 4]
```diff
@@ -39,6 +39,7 @@ class ContinueReturnController @Inject()(
 
   def onPageLoad(period: Period): Action[AnyContent] = cc.authAndGetData(period) {
     implicit request =>
+      println("Got here")
```
[the remark this answers is not in the document]
Comment (oliwiaKot) [reply app/controllers/ContinueReturnController.scala]: done

**Pull request merged**